api/src/main/java/org/openmrs/module/sync2/api/utils/SyncObjectsUtils.java

```diff
@@ -1,43 +1,6 @@
 package org.openmrs.module.sync2.api.utils;
 
-import org.openmrs.module.sync2.api.exceptions.SyncException;
-import org.openmrs.module.sync2.api.model.audit.AuditMessage;
-import org.openmrs.module.sync2.client.rest.resource.Encounter;
-import org.openmrs.module.sync2.client.rest.resource.Location;
-import org.openmrs.module.sync2.client.rest.resource.Observation;
-import org.openmrs.module.sync2.client.rest.resource.Patient;
-import org.openmrs.module.sync2.client.rest.resource.Privilege;
-import org.openmrs.module.sync2.client.rest.resource.Visit;
-
-import static org.openmrs.module.sync2.SyncCategoryConstants.CATEGORY_AUDIT_MESSAGE;
-import static org.openmrs.module.sync2.SyncCategoryConstants.CATEGORY_ENCOUNTER;
-import static org.openmrs.module.sync2.SyncCategoryConstants.CATEGORY_LOCATION;
-import static org.openmrs.module.sync2.SyncCategoryConstants.CATEGORY_OB;
-import static org.openmrs.module.sync2.SyncCategoryConstants.CATEGORY_PATIENT;
-import static org.openmrs.module.sync2.SyncCategoryConstants.CATEGORY_PRIVILEGE;
-import static org.openmrs.module.sync2.SyncCategoryConstants.CATEGORY_VISIT;
-
 public class SyncObjectsUtils {
```

Comment:
Empty Utils class should be deleted

Reply:
completely agree

Reply:
removed at: https://github.com/openmrs/openmrs-module-sync2/pull/58/commits/4d68fa681b44e49afd24eb1b666f7da7a80343e9